apps/fabric-website/src/components/App/App.tsx

```diff
@@ -58,7 +59,11 @@ export class App extends React.Component<IAppProps, any> {
 
   public componentWillReceiveProps(nextProps: IAppProps): void {
     if (nextProps && nextProps.children !== this.props.children) {
-      document.body.scrollTop = document.documentElement.scrollTop = 0;
+      const hash = extractAnchorLink(window.location.hash);
+      // when loading a new component page, we want to scroll to the top
+      if (hash.indexOf('/components') !== -1) {
+        document.body.scrollTop = document.documentElement.scrollTop = 0;
```

Comment:
I don't think you need both of these assignments. You might consider `window.scrollTo(0, 0)` via [Window.scrollTo](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo) or just `document.documentElement.scrollTop` which is `html` DOMNode rather than `body` since the intent is to scroll the entire page.